components/library/pystring/patches/01-Makefile.patch

```diff
@@ -5,7 +5,7 @@
 -LIBDIR ?= /usr/lib
 +PREFIX ?= /usr
 +INCLUDEDIR = ${PREFIX}/include/pystring
-+LIBDIR ?= ${PREFIX}/lib/amd64
++LIBDIR ?= ${PREFIX}/lib/MACH64
```

Comment:
Why about to set `LIBDIR` during build instead of doing this?

Reply:
tried to add: COMPONENT_BUILD_ENV += LIBDIR="/usr/lib/$(MACH64)" in Makefile, but the result is still amd64.

Reply:
What happens when you try to add this to `COMPONENT_BUILD_ARGS` instead?

Reply:
tried: COMPONENT_BUILD_ARGS - same result! - we did the sed approach quite often in the Makefile, so why do want to change it any other?